tests/acceptance/features/webUIMoveFilesFolders/moveFiles.feature

```diff
@@ -78,3 +78,12 @@ Feature: move files
     And file "data.zip" should be listed on the webUI
     And as "user1" file "simple-folder/simple-empty-folder/data.zip" should exist
     But as "user1" file "simple-folder/data.zip" should not exist
+
+    Scenario: move a file to a folder with no change permission
+      Given user "user2" has been created with default attributes
+      And user "user2" has shared folder "simple-folder" with user "user1" with "read" permissions
+      And user "user1" has logged in using the webUI
+      When the user tries to move file "lorem.txt" into folder "simple-folder (2)" using the webUI
+#      Then folder "simple-folder (2)" should be disabled to move files into it
```

Comment:
this commented line not deleted because I would like to know which step actually describes the then step properly. I would like suggestion among this one or the one below(or a new one)